src/Middleware/StaticFiles/ref/Microsoft.AspNetCore.StaticFiles.csproj

```diff
@@ -7,6 +7,7 @@
     <Compile Include="Microsoft.AspNetCore.StaticFiles.netcoreapp3.0.cs" />
     <Reference Include="Microsoft.AspNetCore.Hosting.Abstractions"  />
     <Reference Include="Microsoft.AspNetCore.Http.Extensions"  />
+    <Reference Include="Microsoft.AspNetCore.Routing"  />
```

Comment:
@Tratcher - thoughts? What's the right place for these extension methods to live.

Reply:
See my comments elsewhere. It's not clear you need to bring in the middleware in order to serve a single file.

Reply:
Discussed offline. We're going to go with this for now and evaluate doing some deeper integration between these components.